desktop/envinfo.ts

```diff
@@ -23,6 +23,7 @@ export const getEnvInfo = async () => ({
   node: await getNodeVersion().catch(() => 'unknown'),
   platform: os.platform(),
   arch: os.arch(),
+  isAppImage: process.env.APPIMAGE || '',
```

Comment:
Better to have a flag instead of some custom stuff.
Otherwise, it may become tricky to find all AppImages in the statistics
```suggestion
  isAppImage: !!process.env.APPIMAGE,
```

Reply:
In this [code](https://github.com/spacemeshos/smapp/blob/c3c44f3a6e969413b7cef1ea965a33ce37c84d8a/desktop/main/Networks.ts#L50)

we're combining various string values. And requires only strings

Reply:
Ok...
1. Do you have an idea how we can filter out all AppImages in statistics, using simple filters (starts with / contains / not contains)?
2. I'm not sure that it is a good idea to collect absolute path to AppImage.

So, `Boolean(process.env.APPIMAGE).toString()`?